test_conformance/integer_ops/integer_ops_test_info.h

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (c) 2021 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef INTEGER_OPS_TEST_INFO_H
+#define INTEGER_OPS_TEST_INFO_H
+
+#include "harness/conversions.h"
+
+// TODO: Move this to an even more common location?
```

Comment:
Yeah, that would be good. We've introduced similar templates as part of recent additions to the subgroups suite. It would be good to not reinvent them each time. There might also be a case for adding more type traits and related constructs to the headers directly. I expect this would be useful to applications. We probably shouldn't gate this PR on support landing on the headers though :). For this PR, I think just stating to consolidate these into the harness would already be a very useful first step.

Reply:
I've moved this file to the harness for now.  It's still called `integer_ops_test_info.h` but I think that's OK and we can rename it if/when we add additional traits.

I use a similar file for the bit instructions.  I'm guessing this PR will be merged first - if so, I'll switch it over after this is merged.  We can look at consolidating with the subgroups suite also.

Reply:
Thanks, that's a good first step. As commented [here](https://github.com/KhronosGroup/OpenCL-CTS/pull/1276#pullrequestreview-737483818), I suggest we punt this to a dedicated issue and proceed with this PR as is.